pvanalytics/quality/irradiance.py

```diff
@@ -309,6 +312,9 @@ def check_irradiance_consistency_qcrad(solar_zenith, ghi, dhi, dni,
         value is a dict with keys 'zenith_bounds', 'ghi_bounds', and
         'ratio_bounds' and value is an ordered pair [lower, upper]
         of float.
+    outside_domain : bool, default False
```

Comment:
```suggestion
    outside_domain : bool, default False, optional
```
@kandersolar When labeling a variable as optional, do you still keep the default value?

Reply:
We use `optional` when there is a default value, but the specific value is not really informative or useful to document.  So either document the default value, or call it `optional`, but not both.

> Also, the outside_domain variable need not be bool, it could really be anything, e.g., nan.

Good point.  What can we document for the type then?  `scalar`?

Reply:
I have changed it to scalar